pom.xml

```diff
@@ -39,11 +39,13 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-        <project.build.targetJdk>21</project.build.targetJdk>
-        <air.java.version>21</air.java.version>
+        <project.build.targetJdk>22</project.build.targetJdk>
+        <air.java.version>22</air.java.version>
         <air.check.skip-spotbugs>true</air.check.skip-spotbugs>
         <air.check.skip-pmd>true</air.check.skip-pmd>
         <air.release.preparation-goals>clean verify -DskipTests</air.release.preparation-goals>
+        <dep.trino.version>454</dep.trino.version>
+        <dep.errorprone.version>2.29.2</dep.errorprone.version>
```

Comment:
Please order alphabetically.

Reply:
at first i thought it was wierd that ide does not check this
but i then notice that this `properties` section itself is not ordered

wierd. i dont think sortpom checkstyle doesn't check this at all 
but it saids to sort plugin versions https://github.com/airlift/airbase/blob/master/airbase/pom.xml#L164